common/resource/resourceTest.go

```diff
@@ -218,225 +218,225 @@ func NewTest(
 }
 
```

Comment:
should `resourceTest.go` be removed?

Reply:
Still used by tests I think.

Reply:
No. This contains the actual Test struct that is used by other unit tests. It's not related to the resource struct./
We could change the name of the package. But I'll leave it for now.